src/snap-confine.apparmor.in

```diff
@@ -21,6 +21,8 @@
     /usr/lib/@{multiarch}/libseccomp.so* mr,
     /lib/@{multiarch}/libseccomp.so* mr,
 
+    @{PROC}/@{pid}/auxv r,
+
```

Comment:
1.0.38-0ubuntu0.16.04.10+ppa2 in debian/usr.lib.snapd.snap-confine does not have this rule.